app-backend/common/src/main/scala/S3.scala

```diff
@@ -0,0 +1,37 @@
+package com.azavea.rf.common
+
+import scala.util.{Success, Failure, Try}
+
+import org.apache.commons.io.IOUtils
+
+import com.amazonaws.auth._
+import com.amazonaws.services.s3.{AmazonS3Client, AmazonS3URI}
+import com.amazonaws.services.s3.model.{S3Object, ObjectMetadata}
+
+import java.io._
+import java.net._
+
+package object S3 {
```

Comment:
I think a good article on this topic is this one: http://bguiz.github.io/js-standards/angularjs/application-structure-lift-principle/

Though I think interpretation of the LIFT principle is up to debate -- in this case I'm not sure what should take precedence (locate easily, flat for as long as possible). `common` made sense to me, but in this case maybe a common makes sense later to group similar subprojects? As it is, refactoring and moving things isn't that bad with an IDE and compiler so I don't think we should spend too much time debating this.